build-system-tests/templates/template-aws-exports.js

```diff
@@ -1,3 +1,38 @@
-const awsmobile = {};
+/* eslint-disable */
+
+const awsmobile = {
+  aws_project_region: 'us-east-2',
```

Comment:
Without the mock config, there would be console errors for Auth and in-app-message. For other frameworks, it wouldn't block build, but for *React Native*, since we detect the error message from the develop log. We need the mock config to reduce the console error.